src/main/java/edu/kis/powp/jobs2d/drivers/adapter/drawerJobs2dPatterns.java

```diff
@@ -23,15 +24,19 @@ public void setPosition(int x, int y) {
 
 	@Override
 	public void operateTo(int x, int y) {
+		DrawPanelController controller =
+				DrawerFeature.getDrawerController();
```

Comment:
Constructor injection should be used (using a parameter). This is unnecessary direct coupling with DrawerFeature class.